### PR TITLE
12 2 email pt 2

### PR DESCRIPTION
**Pull request by EUdds**

### CODE REVIEW

File: user-news.md [hunk 1]
```diff
@@ -12,20 +12,37 @@ For older updates not shown on this page, see our [user mailing list
...
 Your CHTC Team
 
+------------------------------------------------------------------------
+## HPC Cluster Queue and Execute Nodes are Down Due to Network Issues
+### Wednesday, December 2, 2020
+
+As of 5:30PM CDT on December 2nd The HPC cluster queue is restored, users are advised to proceed with caution
+~~Greetings,~~
+
+~~*This message is for users of CHTC's HPC Cluster. Users of only the HTC System can ignore.*~~
```
Comment (lmichael107) [inline user-news.md:36]: What are the squiggly symbols?

**Review by lmichael107 — COMMENTED** [review]
squiggly symbols? I didn't see them in prior posts.

**Comment by EUdds** [comment]
That's me striking through the initial email since it no longer applies. Whereas other emails still apply. I can remove them if you would like @lmichael107

File: user-news.md (hunk 1 above)
Comment (EUdds) [reply user-news.md:36]: A strikethrough, it'll look like ~~this~~

**Comment by lmichael107** [comment]
looks good to me!

**Review by lmichael107 — APPROVED** [review]

**Pull request merged**